README.md

```diff
@@ -1,49 +1,48 @@
 # blue-harvest
 
-## Install
+## Getting Started
 You can npm to install the package:
-`npm install blue-harvest --save-dev`,
-or if you are using Yarn,
-`yarn add blue-harvest -D`
+`npm install https://github.com/copperleaftech/blue-harvest`.
```

Comment:
Why is this `copperleaftech/blue-harvest`? This should be changed back. If we need to do a release, we can do so.